public_html/administrator/components/com_jed/models/review.php

```diff
@@ -62,4 +65,81 @@ protected function loadFormData()
 
 		return $data;
 	}
+
+	/**
+	 * Method to get a single record.
+	 *
+	 * @param   integer  $pk  The id of the primary key.
+	 *
+	 * @return  \JObject|boolean  Object on success, false on failure.
+	 *
+	 * @since   4.0.0
+	 */
+	public function getItem($pk = null)
+	{
+		$item = parent::getItem($pk);
+
+		$db    = $this->getDbo();
+		$query = $db->getQuery(true);
+
+		// @TODO calculation for overall_score column
+		$query->select($db->quoteName(
+			[
+				'reviews.published',
+				'reviews.id',
+				'reviews.title',
+				'reviews.created_on',
+				'reviews.ipAddress',
+				'reviews.flagged',
+				'reviews.extension_id',
+				'reviews.created_by',
+				'users.id',
+				'users.username',
+				'extensions.title',
+				'extensions.created_by',
+				'developers.username',
+			],
+			[
+				'published',
+				'id',
+				'title',
+				'created_on',
+				'ipAddress',
+				'flagged',
+				'extension_id',
+				'created_by',
+				'user_id',
+				'username',
+				'extensionname',
+				'developer_id',
+				'developer',
+			]
+		))
+			->from($db->quoteName('#__jed_reviews', 'reviews'))
+			->leftJoin(
+				$db->quoteName('#__users', 'users')
+				. ' ON ' . $db->quoteName('users.id') . ' = ' . $db->quoteName('reviews.created_by')
+			)
+			->leftJoin(
+				$db->quoteName('#__jed_extensions', 'extensions')
+				. ' ON ' . $db->quoteName('extensions.id') . ' = ' . $db->quoteName('reviews.extension_id')
+			)
+			->leftJoin(
+				$db->quoteName('#__users', 'developers')
+				. ' ON ' . $db->quoteName('developers.id') . ' = ' . $db->quoteName('extensions.created_by')
+			)
+			->where($db->quoteName('reviews.id') . ' = ' . (int) $item->id);
+
+		$db->setQuery($query);
+		$data = $db->loadObject();
+
+		$item->developer_id = $data->developer_id;
+		$item->developer = $data->developer;
+		$item->user_id = $data->user_id;
+		$item->username = $data->username;
+		$item->extension_id = $data->extension_id;
+		$item->extensionname = $data->extensionname;
```

Comment:
Codestyle, align the equal signs. If you use PhpStorm with Joomla code style, you can reformat the code and it will automatically align.